test/powershell/Modules/Microsoft.PowerShell.Management/Clipboard.Tests.ps1

```diff
@@ -41,5 +41,10 @@ Describe 'Clipboard cmdlet tests' -Tag CI {
             'world' | Set-Clipboard -Append
             Get-Clipboard -Raw | Should -BeExactly "hello$([Environment]::NewLine)world"
         }
+
+        It 'Set-Clipboard accepts empty string' {
+            '' | Set-Clipboard
```

Comment:
Should we add a test for $null?

Reply:
From PowerShell perspective, they are the same

Reply:
I guess there's no harm in adding it